acceptable-use-policy.md

```diff
@@ -32,7 +33,8 @@ Do not:
  * use Custom Apps to send unsolicited communications, promotions or advertisements, or spam;
  * use Custom Apps to send altered, deceptive or false source-identifying information, including "spoofing" or "phishing";
  * abuse referrals or promotions, for example to get more credits than deserved;
- * sublicense, resell, time share or similarly exploit the Services;
+ * sublicense, resell, time share or similarly exploit the Services, unless otherwise permitted by Retool;
+* use the Services in a manner that Retool determines, in its discretion, circumvents or bypasses fees payable to Retool;
```

Comment:
```suggestion
 * sublicense, resell, time share or similarly exploit the Services, unless otherwise permitted by Retool;
 * use the Services in a manner that Retool determines, in its discretion, circumvents or bypasses fees payable to Retool;
```